src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -87,6 +88,18 @@ if (__DEV__) {
       standardName
     );
 
+    var registrationName = (
```

Comment:
Fixed, I used the formatting you suggested.

[Line 78][78] seems to hint that I should be indenting the lines after the `?` with two spaces:

```js
var registrationName = (
  EventPluginRegistry.possibleRegistrationNames.hasOwnProperty(
    lowerCasedName
  ) ?
    EventPluginRegistry.possibleRegistrationNames[lowerCasedName] :
    null
);
```

Let me know if you prefer this formatting instead and I'll change it.

[78]: https://github.com/ali/react/blob/9ad4be0316d36e446eabd280f680a1b8632323c5/src/renderers/dom/shared/DOMPropertyOperations.js#L78